link.x

```diff
@@ -2,21 +2,32 @@ INCLUDE memory.x
 
 SECTIONS
 {
-  .text ORIGIN(FLASH) :
+  .vector_table ORIGIN(FLASH) :
   {
     /* Vector table */
-    _VECTOR_TABLE = .;
+    _svector_table = .;
```

Comment:
I give up. What does `_pfstack` stand for? 🙂 

I chose _svector_table because of the parallel to _sbss and _sdata which are named after their output section names